packages/devextreme-cli/src/applications/application.angular.js

```diff
@@ -50,19 +51,27 @@ function localPackageExists(packageName) {
 function hasSutableNgCli() {
     return new Promise((resolve, reject) => {
         exec('ng v', (err, stdout, stderr) => {
-            stderr || parseNgCliVersion(stdout).compare(minNgCliVersion) < 0
+            const parsingResult = parseNgCliVersion(stdout);
+            if(globalNgCliVersion === ''
```

Comment:
Maybe change the condition to:
if(!globalNgCliVersion && !stderr)

Reply:
It's a good idea. Done.